sound/soc/intel/boards/sof_cs42l42.c

```diff
@@ -122,7 +122,7 @@ static int sof_cs42l42_hw_params(struct snd_pcm_substream *substream,
 	struct snd_soc_dai *codec_dai = asoc_rtd_to_codec(rtd, 0);
 	int clk_freq, ret;
 
-	clk_freq = 3072000; /* BCLK freq */
+	clk_freq = 2400000; /* BCLK freq */
```

Comment:
This looks good but I am wondering if we can remove the hard-coding and get the frequency directly from the DAI.
I vaguely recall that @keyonjie or @ranj063 added something for the MCLK